examples/tv-app/android/include/account-login/AccountLoginManager.cpp

```diff
@@ -102,3 +103,15 @@ void AccountLoginManager::GetSetupPin(char * setupPin, size_t setupPinSize, cons
     }
     ChipLogProgress(Zcl, "Returning pin for content app for endpoint %d", mEndpointId);
 };
+
+uint16_t AccountLoginManager::GetClusterRevision(chip::EndpointId endpoint)
+{
+    if (endpoint >= EMBER_AF_CONTENT_LAUNCHER_CLUSTER_SERVER_ENDPOINT_COUNT)
+    {
+        return mDynamicClusterRevision;
+    }
+
+    uint16_t clusterRevision = 0;
+    Attributes::ClusterRevision::Get(endpoint, &clusterRevision);
```

Comment:
`::Get` returns a status that probably should be handled (at least log on failure). Applies throughout the PR.

Reply:
Updated.